### PR TITLE
Added support for Int64

### PR DESCRIPTION
**Pull request by YarnoVdW**
The swagger_parser lacked support for Int64, which is crucial for a project I'm working on. I've added the implementation and a test to make sure everything works as expected.

### CODE REVIEW

File: swagger_parser/test/generator/data_classes_test.dart [hunk 1]
```diff
@@ -2759,4 +2759,43 @@ data class ClassName(
...
     });
   });
+
+  test('int64 to bigint ', () async {
+    const dataClass = UniversalComponentClass(
+      name: 'ClassName',
+      imports: {},
+      description: 'Test class',
+      parameters: [
+        UniversalType(type: 'integer', format: 'int64', isRequired: false),
```
Comment (Carapacik) [inline swagger_parser/test/generator/data_classes_test.dart:2769]: Plz add name for a type to avoid the name "null"

Comment (YarnoVdW) [reply swagger_parser/test/generator/data_classes_test.dart]: Done! :)

**Review by Carapacik — APPROVED** [review]

**Comment by YarnoVdW** [comment]
@Carapacik FYI: The json_serializable package cannot handle BigInts properly so this push can potentially be breaking. I've downgraded back because I can't seem to get it working with the json_serializable package (and also with retrofit_generator)

**Comment by Carapacik** [comment]
BigInt works fine with the library, at least with the latest version()json_serializable: ^6.8.0)

**Comment by YarnoVdW** [comment]
Hmmm; because when I use it it does a BigInt.parse(value as String); But my value is an int, so the "as String" breaks it for me

**Comment by Carapacik** [comment]
Ok. I will revert this commit

**Pull request merged**